includes/data-port/class-sensei-data-port-utilities.php

```diff
@@ -99,12 +100,13 @@ public static function get_attachment_from_source( $source, $parent_id = 0 ) {
 	 * downloaded file. If the file has been already downloaded an linked to an attachment, it returns the existing
 	 * attachment instead.
 	 *
-	 * @param string $external_url  The external url.
-	 * @param int    $parent_id     The attachment's parent id.
+	 * @param string $external_url       The external url.
+	 * @param int    $parent_id          The attachment's parent id.
+	 * @param array  $allowed_mime_types Allowed mime types.
 	 *
 	 * @return int|WP_Error  The attachment id or an error.
 	 */
-	public static function create_attachment_from_url( $external_url, $parent_id = 0 ) {
+	public static function create_attachment_from_url( $external_url, $parent_id = 0, $allowed_mime_types = null ) {
```

Comment:
Another check below before returning `$existing_attachment[0]`

Reply:
Also fixed with a bunch of commits ;)